.github/workflows/_elixir.yml

```diff
@@ -160,40 +98,10 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - uses: actions/checkout@v4
-      - name: Tool Versions
-        id: versions
-        uses: marocchino/tool-versions-action@v1.2.0
-      - uses: erlef/setup-beam@v1
-        id: setup-beam
+      - uses: ./.github/actions/setup-elixir
         with:
-          otp-version: ${{ steps.versions.outputs.erlang }}
-          elixir-version: ${{ steps.versions.outputs.elixir }}
-      - uses: actions/cache/restore@v4
-        name: Restore Elixir Deps Cache
-        id: cache
-        env:
-          cache-name: elixir-deps-${{ env.MIX_ENV }}
-        with:
-          path: |
-            elixir/deps
-            elixir/_build/${{ env.MIX_ENV }}
-          key: ubuntu-22.04-${{ runner.arch }}-${{ steps.setup-beam.outputs.elixir-version }}-${{ env.cache-name }}-${{ hashFiles('elixir/mix.lock') }}
-      - name: Install Dependencies
-        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
-        run: mix deps.get --only $MIX_ENV
-      - name: Compile Dependencies
-        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
-        run: mix deps.compile --skip-umbrella-children
-      - uses: actions/cache/save@v4
-        name: Save Elixir Deps Cache
-        if: ${{ github.ref_name == 'main' }}
-        env:
-          cache-name: elixir-deps-${{ env.MIX_ENV }}
-        with:
-          path: |
-            elixir/deps
-            elixir/_build/${{ env.MIX_ENV }}
-          key: ubuntu-22.04-${{ runner.arch }}-${{ steps.setup-beam.outputs.elixir-version }}-${{ env.cache-name }}-${{ hashFiles('elixir/mix.lock') }}
+          mix_env: ${{ env.MIX_ENV }}
+          force_compile: true
```

Comment:
I don't see a `force_compile` param in `setup-elixir` action. Also, since we can't use subfolders should we name those steps differently? Like `_elixir-setup.yml` so that they are next to each other when browsed?

Reply:
`force_compile` removed.

> Like _elixir_fetch-and-cache-deps.yml so that they are next to each other when browsed?

Hm, not sure I'm following. Like split this composite into two for better maintainability?

Reply:
@jamilbk I meant that we can't use subfolders so we can at least name it by some convention so that in the file system `_elixir_*` stuff is grouped together. But this is not important.